requirements.txt

```diff
@@ -1,4 +1,4 @@
-marshmallow==3.0.0b9
+marshmallow>=3.0.0b12,<=3.0.0b18
```

Comment:
this should probably be a hard pin

Reply:
hard pin txt file since this is installed as "app" (ie for dev and testing) and soft pin setup (installed as lib)